sync/Cargo.toml

```diff
@@ -29,5 +29,6 @@ ethcore-ipc-nano = { path = "../ipc/nano" }
 parking_lot = "0.2.6"
 
 [features]
-default = []
+default = ["ipc"]
```

Comment:
Probably best not to have this as default; it'll already get triggered by parity's default features and IPC probably isn't wanted when testing as it means more work for rustc